.github/workflows/ci.yml

```diff
@@ -0,0 +1,66 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Cryptol and Saw Python Clients
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8
+        pip install -r requirements.txt
+    # - name: Lint with flake8
```

Comment:
It's unclear if you want to land this now to have some CI working, or if you want to uncomment everything before unmarking this as a WIP. I'd be fine with either approach, but if you leave things uncommented, it might be worth opening issues for each thing not in CI and referencing them from the appropriate locations here.

Reply:
I'll probably try a little tinkering and see if I can't just get everything at a good baseline with the initial merge of the PR, and make issues if there are remaining tasks for later.